app/models/container.rb

```diff
@@ -39,6 +42,10 @@ def event_where_clause(assoc = :ems_events)
 
   PERF_ROLLUP_CHILDREN = nil
 
+  def display_name
+    "#{container_group.name} / #{name}"
```

Comment:
@yaacov I don't think we should have spaces here. (Probably they're nicer in the UI but they feel wrong looking at this method).